test/exercism/track_stream_filters_test.rb

```diff
@@ -77,37 +82,36 @@ def test_filters
 
     assert_equal 'Anagram', item2.text
     assert_equal '/tracks/go/exercises/anagram', item2.url
-    assert_equal 1, item2.total
-    assert_equal 1, item2.unread
+    assert_equal 2, item2.total
+    assert_equal 2, item2.unread
     refute item2.active?
 
     assert_equal 'Triangle', item3.text
     assert_equal '/tracks/go/exercises/triangle', item3.url
-    assert_equal 1, item3.total
-    assert_equal 1, item3.unread
+    assert_equal 2, item3.total
+    assert_equal 2, item3.unread
     refute item3.active?
 
-    filter = TrackStream::ViewerFilter.new(bob.id, 'go', true)
+    only_mine = true
+    filter = TrackStream::ViewerFilter.new(bob.id, 'go', only_mine)
     assert_equal 1, filter.items.size
 
-
     item = filter.items.first
 
-
     assert_equal 'My Solutions', item.text
     assert_equal '/tracks/go/my_solutions', item.url
-    assert_equal 2, item.total
+    assert_equal 3, item.total
+    assert_equal 2, item.unread
```

Comment:
Are you inverting read/and unread? Only the views by the person who is looking at it should count.

Bob has viewed `triangle` and not viewed the two others (so 2 unread).